data_science_colleges.csv

```diff
@@ -340,6 +346,8 @@ University of Pittsburgh-Pittsburgh Campus,MSIS - Big Data Analytics,M,T,T,US,PA
 Villanova University,Graduate Certificate in Applied Statistics,C,F,T,US,PA,Villanova,Liberal Arts and Sciences,http://www1.villanova.edu/content/dam/villanova/artsci/gradstudies/AppliedStatisticsGraduateInformationSheet.pdf
 Villanova University,Master of Science in Applied Statistics,M,F,T,US,PA,Villanova,Liberal Arts and Sciences,http://www1.villanova.edu/villanova/artsci/mathematics/academic-programs/graduate/programs/msapplied.html
 Villanova University,Master of Science in Analytics,M,T,F,US,PA,Villanova,School of Business,http://www1.villanova.edu/villanova/business/graduate/specializedprograms/msa/programinfo.html
+Villanova University,Online Master of Business Administration,M,T,F,US,PA,Villanova,School of Business,http://taxandbusinessonline.villanova.edu/master-of-business-administration
+Villanova University,Online Master of Science in Analytics,M,T,F,US,PA,Villanova,School of Business,http://taxandbusinessonline.villanova.edu/master-of-science-in-analytics
```

Comment:
this program is already on the list, just using a different URL
the URL here is better though